### PR TITLE
Freeze MENDER_ACCEPTANCE_URL 

### PR DESCRIPTION
**Pull request by oleorhagen**
mender-image-tests packaged cherry

### CODE REVIEW

**Comment by oleorhagen** [comment]
This currently blocks: https://github.com/mendersoftware/mender-convert/pull/265

File: scripts/test/test-utils.sh [hunk 1]
```diff
@@ -134,10 +136,10 @@ run_tests() {
 }
 
 get_pytest_files() {
-  wget -N ${MENDER_ACCEPTANCE_URL}/pytest.ini -P $WORKSPACE/mender-image-tests
```
Comment (lluiscampos) [inline scripts/test/test-utils.sh:137]: I suggest we just modify `MENDER_ACCEPTANCE_URL` to something like `https://raw.githubusercontent.com/mendersoftware/meta-mender/c791015fb14f27b7b79143494ee50e42f70c4661/tests/acceptance` (last commit before the refactoring.

The reason is that 2.2.x is not prepared to execute the runtime tests (`test_update.py`) and it is not worth the effort.

**Review by lluiscampos — CHANGES_REQUESTED** [review]
I suggest a completely different (and simple) workaround. See comment below.

**Review by lluiscampos — APPROVED** [review]
Thanks. Please adjust the PR tittle :)

**Pull request merged**